src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -3075,7 +3077,7 @@ TEST_P(conv_fp32_activation_exp_onednn, basic) {
         reorder("reorder_bfyx", "activation", p.default_format, data_types::f32)
     );
 
-    tolerance = 1e-2f;
+    tolerance = default_tolerance(p.default_type);
```

Comment:
why did you change this? It is decreased from 1e-2 to 1e-5

Reply:
Testcases below are changed from 1e-2 to default_tolerance(p.default_type) and those tests are all passed.

Reply:
conv_fp32_activation_abs_onednn
conv_fp32_activation_mish_onednn
conv_fp32_activation_swish_onednn
conv_fp32_activation_hswish_onednn
conv_fp32_activation_exp_onednn